src/user/dto/CreateUser.dto.ts

```diff
@@ -15,7 +15,9 @@ export class createUserDTO {
   @MinLength(8)
   password: string;
   @IsNotEmpty()
+
```

Comment:
Empty lines are between fields, not between decorator and variable.

Reply:
done